tests/e2e/test_e2e_images.py

```diff
@@ -200,19 +200,24 @@ def test_images_delete(
     tag: str,
 ) -> None:
     image_no_tag = image.replace(f":{tag}", "")
```

Comment:
I think problem can be because image is not uploaded to registry. `image` fixture only creates local image.

Reply:
Thanks! I somehow assumed `image` does the push, but never checked it.